core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -78,6 +78,12 @@ jQuery.extend( jQuery.fn.dataTableExt.oSort, {
     }
 } );
 
+jQuery.extend( jQuery.fn.dataTableExt.ofnSearch, {
+    "appid-numeric": function ( a ) {
+        return a.replace(/[\r\n]/g, " ").replace(/<.*?>/g, "");
```

Comment:
CC @ajbozarth for questions of JS style. There isn't a nice built-in way to strip HTML? maybe not. I imagine this does work correctly for well-formed HTML